requirements.txt

```diff
@@ -1,4 +1,6 @@
-mkdocs<=0.16
-python-markdown-math
+doctr
```

Comment:
I wonder if this shouldn't be in the requirements.txt, but rather somewhere else (requirements-test.txt, or requirements-ci.txt)?

Reply:
That makes sense for a package but I'm not sure we need the extra complexity for a doc page. Also, doctr is not only used in the CI, every now and then I need to issue some commands locally to update the config and/or test.